chain/gen/genesis/genesis.go

```diff
@@ -332,8 +332,32 @@ func createAccount(ctx context.Context, bs bstore.Blockstore, cst cbor.IpldStore
 			return xerrors.Errorf("failed to create empty map: %v", err)
 		}
 
+		var signers []address.Address
+
+		for _, e := range ainfo.Signers {
+			idAddress, _ := keyIDs[e]
+			// Check if actor already exists
+			_, err := state.GetActor(e)
+			if err == nil {
+				continue
+			}
+			st, err := cst.Put(ctx, &account.State{Address: e})
+			if err != nil {
+				return err
+			}
+			err = state.SetActor(idAddress, &types.Actor{
```

Comment:
If an actor corresponding to ID doesn't exists, add an empty actor with zero balances.